universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -143,6 +169,78 @@ private static Question applicantHouseholdMemberName(QuestionEnum ignore) {
     return maybeSave(definition);
   }
 
+  private static Question applicantName(QuestionEnum ignore) {
+    QuestionDefinition definition =
+        new NameQuestionDefinition(
+            VERSION,
+            "applicant name",
+            Path.create("applicant.applicant_name"),
+            Optional.empty(),
+            "name of applicant",
+            LifecycleStage.ACTIVE,
+            ImmutableMap.of(Locale.US, "what is your name?"),
+            ImmutableMap.of(Locale.US, "help text"));
+    return maybeSave(definition);
+  }
+
+  private static Question applicantPets(QuestionEnum ignore) {
+    QuestionDefinition definition =
+        new CheckboxQuestionDefinition(
+            VERSION,
+            "checkbox",
+            Path.create("applicant.checkbox"),
+            Optional.empty(),
+            "description",
+            LifecycleStage.ACTIVE,
+            ImmutableMap.of(Locale.US, "question?"),
```

Comment:
(blocking) please put a real question here. It might help people using questions decide which ones to use.